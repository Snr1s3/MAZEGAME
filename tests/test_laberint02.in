gAAAAABmL4Ajed-elXLtiWhiu1fZr9UwqJ9g6hPXwyWp4JrhYUoKbkf5k6xs1hdTYf-IH80H2Fa6XngEHVPU0RYqvGCj1lJXoQ==