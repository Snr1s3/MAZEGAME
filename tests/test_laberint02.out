gAAAAABmL4Ajcyy3Tgo5AgRmPJzvHfVvudoHYy0NoZbrle4VkHoGvvKyw6OF-K5eEb_mVxfX9I2Rl8BMjBNi-goFTUWzgIKmIfPQqGZZwzccmtaO0MR4zfkgpwkP1p9eUBHGcAQpJ_5xLQnqwjRhM-975ZU69IaA-jsZWgjsDEM1FqR1uexE6a80TZ1qeV3yAptZ2HBID5KBJzNC0z8byxKJdvpTPkIJdMH_mKsVbOMee98ogCIXyTFB4iJWyQAWqI2dQ4PRvavXOTUv0XDhyoA6tIqiAPcXmbl7fZP3xFjGQLKK6SC55S6BF_fJSjFpNP4ov_yQqCq4U3GAi6IYVWxBtw5P5osgtmcvP65114Wa6dRWyupj84myiN5SXjYVsoGjcuBotW_cL8EZq3HmcJbdWx6y0hGPyBGxrr_iLtXyDxx4mrkv8K8Ax9Ht3ZcnAai2DrCoGGKWnmI45btQJojO7698B3VCPJJZ4o-Dqa0ge79DJTBK1cvtymxcge8x3RAPVrHb-8QEoj8irDuC0nebYWSfOTuEFW_6ogrnTljtWTWXPlDhifuhiC8owyvUQyp0X1c30Tu7_5IwTck8n6wxeugH7Iltf2gaSsDvg-ddVEgF-KoYdwg=